### DISCUSSION TITLE
MinRunners option in latest helm chart version does not seem to work

### DISCUSSION THREAD

**Discussion by pdonorio** [actions/actions-runner-controller#3361] [Questions] [answered]
Hi 👋 we were testing the latest ARC runners helm chart 
which is https://github.com/actions/actions-runner-controller/releases/tag/gha-runner-scale-set-0.8.3 at the time of writing.

We found that after upgrading to it from 0.8.0
the MinRunners didn't work correctly (e.g. we set it to 10 but it was 0 when nothing was running)
and that the time to start a new pod was longer (e.g. 1 min instead of seconds).

The wrong behavior has been observed also after downgrading to 0.8.2, so we went back to 0.8.0.

We just set things like
```yaml
minRunners: 10
maxRunners: 20
```

in the `values.yaml` file added to the helm release.

Is this expected?

**Comment by pdonorio** [comment]
bump 🙏

**Reply by casey-robertson-paypal** [reply]
we are running 0.8.3 and haven't seen this issue.  If you shared your chart values or a rendered helm template maybe we can see what's going on.

**Reply by pdonorio** [reply, marked as answer]
0.9.1 solved this, it seems. tested today
